pkg/controller/controller_instance.go

```diff
@@ -339,6 +433,18 @@ func (c *controller) reconcileServiceInstanceAdd(instance *v1beta1.ServiceInstan
 		return nil
 	}
 
+	// don't DOS the broker.  If we already did a provision attempt that ended with a non-terminal
+	// error then we set a next retry time.  Observe that.
```

Comment:
more context for "observe that".

Reply:
I'll update.  @MHBauer thanks for the review and comments, I'll update accordingly.